src/main/resources/view/MeetingListCard.fxml

```diff
@@ -27,11 +27,11 @@
         <Label fx:id="message" text="\$first" wrapText="true" styleClass="cell_big_label"/>
       </HBox>
       <HBox spacing="5" alignment="CENTER_LEFT">
```

Comment:
Past meetings should be "greyed out" but this doesn't seem to be the case in the light theme

Reply:
Styling issues should be fixed now. Let me know if there's still anything missing for the styling!